### PR TITLE
5921 - Disable "Create Backup" if backups are disabled 

### PR DESCRIPTION
**Pull request by gaurav061**

### CODE REVIEW

File: managed/ui/src/components/tables/ListBackups/ListBackups.js
Comment (yb-andrew) [inline managed/ui/src/components/tables/ListBackups/ListBackups.js:411]: - You can access an Object property with a simple object accessor like so `currentUniverse.universeConfig.takeBackups`. Use brackets only if passing a variable or number.
- Don't need the ternary to set to `true` or `false`, it is implied.
- Make sure you check that `currentUniverse.universeConfig` is true before trying to access `takeBackups` property.

**Review by yb-andrew — CHANGES_REQUESTED** [review]

File: managed/ui/src/components/tables/ListBackups/ListBackups.js
Comment (sshev) [reply managed/ui/src/components/tables/ListBackups/ListBackups.js:411]: I think a good idea would be to change `takeBackups` to boolean type, not sure why it's a string

Comment (gaurav061) [reply managed/ui/src/components/tables/ListBackups/ListBackups.js:411]: @yb-andrew regarding second point using ternary operator, As ```takeBackups``` is of type string. I will address other points and raise the PR

**Comment by gaurav061** [comment]
Please take a look at the current change. I have addressed the comments, Please acknowledge.

**Review by yb-andrew — APPROVED** [review]
Approving after you remove redundant ternary operator.

File: managed/ui/src/components/tables/ListBackups/ListBackups.js
Comment (sshev) [inline managed/ui/src/components/tables/ListBackups/ListBackups.js:411]:
```suggestion
                      disabled={currentUniverse.universeDetails.backupInProgress || currentUniverse.universeConfig?.takeBackups === "false"}
```

Comment (gaurav061) [reply managed/ui/src/components/tables/ListBackups/ListBackups.js]: I have removed the redundant code.

**Review by yb-andrew — APPROVED** [review]
LG, thanks!

**Review by sshev — APPROVED** [review]

**Comment by sshev** [comment]
Change PR title to `[#5921] Platform: disable "Create Backup" if backups are disabled` before merging, please.
It should follow the pattern `[#1234, #1235] area: short title`

**Pull request merged**
